examples/PalmerPenguins.jl

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+using Pkg; Pkg.activate("../docs")
+using PalmerPenguins, Plots, DataFrames, Random, StatsPlots
+using KernelFunctions, AbstractGPs, GPLikelihoods
+using EllipticalSliceSampling, Distributions
+df = dropmissing(DataFrame(PalmerPenguins.load()))
+df = df[randperm(nrow(df)), :]
+df[!,:flipper_length_mm] = convert(Vector{Float64}, df[!,:flipper_length_mm])
+df[!,:body_mass_g] = convert(Vector{Float64}, df[!,:body_mass_g])
+first(df, 5)
+
+# ## Binary classification using Bernoulli likelihood
+
+unique(df[!,:sex])
+
+
+x = RowVecs(Array(df[:, [
+                :bill_length_mm, 
+                :bill_depth_mm, 
+                :flipper_length_mm, 
+                :body_mass_g
+                ]]))
+y = df[!,:sex] .== "female"
+scatter(
+    df[!, :bill_length_mm],
+    df[!, :bill_depth_mm],
+    df[!, :flipper_length_mm],
+    group = df[!, :sex],
+    m = (0.5, [:+ :h :star7], 5),
+)
+
+x_train, y_train = x[1:266], y[1:266]
+x_test, y_test = x[267:end], y[267:end];
+
+
+# logpdf without any parameters
+
+k = SqExponentialKernel()
+f = LatentGP(GP(k), BernoulliLikelihood(), 0.001)
```

Comment:
This is not consistent with what is used during inference:
```suggestion
f = LatentGP(GP(k), BernoulliLikelihood(), 0.01)
```